snooty/rstspec.toml

```diff
@@ -590,8 +592,8 @@ inherit = "_guides-base"
 
 [directive.seealso]
 help = """Related pages. Typically a list."""
-inherit = "_guides-base"
-# example = """.. seealso::
+inherit = "admonition"
```

Comment:
I think this seealso is guides-specific, rather than the sphinx-y `seealso` directive. This one should maybe be scoped to guides only and then have a separate `seealso` that's like the `see`?

Reply:
Good point, there is a `seealso` that's used everywhere so I assumed they were the same. I will add one with the rest of the admonitions and revert this to its previous config.